com/williamfiset/algorithms/sorting/ShellSort.java

```diff
@@ -0,0 +1,60 @@
+// Java implementation of ShellSort 
```

Comment:
Can you add this shell sort to the readme?

https://github.com/williamfiset/Algorithms/blob/master/README.md

Reply:
yes , i can add the shell sort to readme . Is there anything else that i am supposed to do?

Reply:
I saw you submitted another PR for this. Can you add the README in this PR so that all the changes are together? Thanks!

Reply:
ok , i will do.